src/picologging/__init__.py

```diff
@@ -26,6 +26,7 @@
     _checkLevel,
     StringTemplateStyle,
     BufferingFormatter,
+    makeLogRecord,
```

Comment:
Why is `makeLogRecord()` imported? That function shouldn't be called anywhere as it doesn't make a `LogRecord` instance from the pico logging module, it's also really slow (and one of the reasons pico logging is quicker)

Reply:
Yes you're right, but it's only used in the tests.
And the reason I dide expose it was that logging module does this.
Maybe we can just import it in the tests and avoid it here.

Reply:
Or we can just create this in `picologging` it's easier.

Reply:
Done.